### PR TITLE
Adding library

### PR DESCRIPTION
**Pull request by Antonio-Laguna**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export default class Metrics {
+  constructor(host, debug = false) {
+    this.host = host;
+    this.debug = debug;
+
+    if (this.host[host.length - 1] === '/') {
```
Comment (jgallen23) [inline index.js]: what is this for?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      .join(',');
+
+    const url = `${this.host}/t.gif?type=${type}&value=${value}&tags=${tags}`;
+    this.log(url);
+    const img = new Image();
+    img.src = url;
+    document.body.appendChild(img);
+  }
+
+  trackLoad() {
```
Comment (jgallen23) [inline index.js]: lets pull this out into a separate file.  not all sites will use this

File: index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+export default class Metrics {
+  constructor(host, debug = false) {
```
Comment (jgallen23) [inline index.js]: lets do debug as localStorage like domodule

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Antonio-Laguna) [reply index.js]: So it doesn't matter whether we do `localhost` or `localhost/`

Comment (jgallen23) [reply index.js]: k, thats fine to leave then

**Pull request merged**